test_pull_requests

```diff
@@ -768,8 +802,8 @@ popd
       trigger_time = nil
       trigger_login = nil
       if pull_request['title'] =~ trigger_regex || pull_request['body'] =~ trigger_regex
-        if user_trusted?(login, repo, settings)
-          # Once we determine that the trusted user has a trigger statement in their pull
+        if user_in_group?(login, repo, settings, group_identifier)
+          # Once we determine that the grouped user has a trigger statement in their pull
```

Comment:
I probably would have left the term trusted as the generic name and used test_trusted and admin_trusted to separate the more specific methods.